lib/hanami/routing/route.rb

```diff
@@ -50,7 +50,8 @@ def generate(resolver, options = {}, &endpoint)
       # @since 0.2.0
       # @api private
       def nested_router
-        dest.routes if dest.respond_to?(:routes)
+        dest.routes if dest.respond_to?(:routes) &&
```

Comment:
@cyphactor At the next line there is `dest.routes` invocation. If we remove this `respond_to?` check, it can crash for object that don't respond to `#routes`. :wink: